### ISSUE TITLE
Support Password Protecting Web UI

### ISSUE THREAD

**Issue by Alexqw** [volumio/Volumio2#375]
Sorry for the deluge of new tickets. ;-)

Node.js apparently doesn't support using Apache-style .htaccess files (useful for adding Basic Auth). I have no experience developing in node, but from what I read, there are only programmatic solutions for adding Basic Auth.

So, thus I opened this feature request, hoping for a simple password prompt in order to access the web UI.

---Alex

**Comment by volumio** [comment]
Yes, it's on our to do list... It's a frontend issue anyway... We'll be using some angular or express middleware

**Comment by Alexqw** [comment]
@volumio Excellent. :-)

Is this todo list available publicly? I didn't mean to open a bunch of tickets for things which are already known todos in a pre-release project. :-/

---Alex

**Comment by fooness** [comment]
Dear developers, are there any news concerning this?

**Comment by fooness** [comment]
I think this would be a real important feature. The last days I tried to look into the code, but I’m absolutely clueless on how to work with those websockets. Hope this feature will very soon exist.

Thank you in advance!

**Comment by nyanloutre** [comment]
Would like to use this feature too, as my computer will be connected to a non trusted network (students residence)

This aside, awesome work :)

**Comment by BeCracked** [comment]
I have the same issue (students residence) which is why I want to move on from the chromecast, which I am currently using, because it basicly broadcasts a "control me" to EVERY Android device in the network.
So requiring at least a simple password for accessing the frontend would make me feel a lot better, especially when you have logins openly visible in the frontend.

**Comment by whatsnewsisyphus** [comment]
I need this too, similarly sharing a school network. Let me know if I could help in any way.

**Comment by silelmot** [comment]
Is there smth new? I need this for a school too

**Comment by alexnoyes** [comment]
Is there any update on this Feature Request?

**Comment by pburndorfer** [comment]
Any updates concerning password protection?

**Comment by Aido21** [comment]
hey
any news? 7 years passed
